src/tests/ftest/util/MpioUtils.py

```diff
@@ -43,12 +43,11 @@ def mpich_installed(self, hostlist):
             checkmpich = subprocess.check_output(["ssh", hostlist[0], "which mpichversion"])
 
             # Obtaning the location where mpich is installed by removing "mpichversion"
-            self.mpichinstall = checkmpich[:-13]
-
+            self.mpichinstall = checkmpich[:-17]
             return True
 
-        except subprocess.CalledProcessError as e:
-            print ("Mpich not installed \n {}".format(e))
+        except subprocess.CalledProcessError as error:
+            print ("Mpich not installed \n {}".format(error))
```

Comment:
(pylint-superfluous-parens) Unnecessary parens after 'print' keyword

Reply:
maybe because of the space after print?  since you had the __future thing at the top.